src/GitVersion.Core/MergeMessage.cs

```diff
@@ -97,13 +97,16 @@ private ReferenceName GetMergedBranchName(string mergedBranch)
         mergeCommit.NotNull();
         configuration.NotNull();
 
-        mergeMessage = null;
+        mergeMessage = new(mergeCommit.Message, configuration);
```

Comment:
Can you reverse this so it initializes to `null` by default and only to a `MergeMessage` instance in the case `isValidMergeCommit` is `true`?

Reply:
@asbjornu sure, I've added an additional variable to check the mergedBranch.